spec/features/creating_an_employment_appeal_tribunal_decision_spec.rb

```diff
@@ -88,24 +88,27 @@
   context 'in production' do
     before do
       allow(Rails.env).to receive(:development?).and_return(false)
-      log_in_as_editor(:gds_editor)
+      stub_any_publishing_api_call
+      publishing_api_has_item(research_output)
     end
 
     context "when logged in as an editor" do
-      before { log_in_as_editor(:editor) }
+      before { log_in_as_editor(:cma_editor) }
```

Comment:
seems like possibly this should be moved into a different test/renamed test, since its in a namespace for EAT stuff